docs/manual/developer_guide.adoc

```diff
@@ -594,16 +600,17 @@ The <tt>/tmp</tt> partition is used as temporary storage by many programs. Placi
 * `description`: Human-readable HTML description, which provides broader context for non-experts than the rationale. For example, description of the `partition_for_tmp` rule states that:
 +
 The <tt>/var/tmp</tt> directory is a world-writable directory used for temporary file storage. Ensure it has its own partition or logical volume at installation time, or migrate it using LVM.
-* `ocil`: Defines assert statements. The `clause` attribute contains the statement, and the element text describes how to determine whether the statement is true or false. Check out rule `encrypt_partitions` in `shared/xccdf/system/software/disk_partitioning.xml`, that contains the `partitions do not have a type of crypto_LUKS` clause.
+* `ocil`: Defines asserting statements to check whether or not the rule is valid.
+* `ocil_clause` attribute contains the statement, and the element text describes how to determine whether the statement is true or false. Check out `encrypt_partitions.rule` in `linux_os/guide/system/software/disk_partitioning/`: this contains a `partitions do not have a type of crypto_LUKS` `ocil_clause`.
 
 A rule may contain those reference-type elements:
```

Comment:
I think it mixes elements, attributes, XML and YAML. I'd suggest explicitly defining that this section describes YAML structure and then adjust the rest of text accordingly.